Formula/gffread.rb

```diff
@@ -0,0 +1,32 @@
+class Gffread < Formula
+  desc "GFF/GTF format conversions, region filtering, FASTA sequence extraction"
+  homepage "https://github.com/gpertea/gffread"
+  url "https://github.com/gpertea/gffread/releases/download/v0.12.7/gffread-0.12.7.tar.gz"
+  sha256 "bfde1c857495e578f5b3af3c007a9aa40593e69450eafcc6a42c3e8ef08ed1f5"
+  license "MIT"
+
+  resource "gclib" do
+    url "https://github.com/gpertea/gclib/archive/refs/tags/v0.12.7.tar.gz"
+    sha256 "9c74db64c4ee1fb93a6c02fa227258baafc337303f26ce3ba0cf37dfc1ba687f"
+  end
+
+  def install
+    resource("gclib").stage do
+      system "make"
+    end
```

Comment:
Are you sure this resource is needed? The staging directory is deleted when this block ends, so whatever `make` built isn't available in `buildpath`.

Reply:
yeah, that is right, the gclib is already bundled in the release tarball, let me remove it.